Makefile

```diff
@@ -47,34 +47,22 @@ install: # [Go package] - # Build and install everything, or only the specified
 	go install -v -ldflags="$(GO_LDFLAGS)" $(PACKAGE)
 .PHONY: install
 
-install-all: install install-tools install-tools-linting install-tools-testing # Install everything for and of this repository.
+install-all: install install-tools # Install everything for and of this repository.
```

Comment:
Plz check if the removed make targets are mentioned in the README.md.

Reply:
Only `make install-all`